tests/test_media_multipart.py

```diff
@@ -291,7 +291,7 @@ def test_body_part_properties():
         if part.content_type == 'application/json':
             assert part.name == part.name == 'document'
         elif part.name == 'file1':
-            assert part.filename == part.filename == 'test.txt'
+            assert part.filename == 'test.txt'
```

Comment:
I think this might not be oversight as it looks like. I don't remember now, but maybe I want to make sure that this property can be referenced multiple times?

Reply:
I can revert but we should likely leave a note mentioning why it's like this

Reply:
Yes, I agree, we need to add a note.